scripts/teardown.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Comment:
Should we add `set -e` right after the shebang? (Fail fast if any command fails)

Reply:
ah, wasn't aware of that command

let me read up on that and add